mac-runner/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+# chia-runner/tasks/main.yml
+- name: Clone runner code and unzip
+  shell: mkdir actions-runner && cd actions-runner && curl -O -L https://github.com/actions/runner/releases/download/v2.273.5/actions-runner-linux-x64-2.273.5.tar.gz && tar xzf ./actions-runner-linux-x64-2.273.5.tar.gz
```

Comment:
Same comment as above - would be nice to break this up into the ansible tasks that are designed for directories, donwloading files, etc